analyzers/tests/SonarAnalyzer.UnitTest/Rules/DeadStoresTest.cs

```diff
@@ -36,21 +36,36 @@ public class DeadStoresTest
 
         [TestMethod]
         public void DeadStores_SonarCfg() =>
-            sonarCfg.AddPaths("DeadStores.SonarCfg.cs").Verify();
+            sonarCfg.AddPaths("DeadStores.SonarCfg.cs")
```

Comment:
NITPICK:  the .Verify() invocation should go to the same line. Simple test case where it  contains only addpaths and verify we tend to put on the same line.
I am not sure it is documented anywhere (If it is not in the coding style than we should most probably document it)

Reply:
Applies for the whole PR.